docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md

```diff
@@ -41,6 +41,9 @@ Note there is no need to add the Play `evolutions` component to your dependencie
 
 > **Note**: You can see th
 
+#### Running evolutions with Slick using compile-time DI
+If you are using Slick, you will also need to mix in the `SlickEvolutionsComponents` trait in addition to the `EvolutionsComponents` trait to `ApplicationComponents`. Else evolutions will not recognize the Slick database configurations in the configuration files.
+
```

Comment:
Actually, what do you think about changing the name to `SlickDBComponents` instead of `SlickEvolutionsComponents`? That would make things more straightforward since this basically replaces `DBComponents`. Then we can deprecate the whole trait instead of just that `api` method.